### ISSUE TITLE
[BUG] Heading in header component should not be a <h1>

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#114]
Best practice is to use one unique h1 per page that describes what that page is about, normally the same as the page title. That h1 preferably starts just above the main content. This way it can also be used by user as a way to skip to the main content of the page.

Headings create an outline for the page, similar to a term paper outline or table of contents. The <h1> describes the page as a whole (and should be similar /same as the page <title>). A page should typically have only one <h1>.(https://webaim.org/techniques/semanticstructure)

In my opinion the Heading should be changed to a <h2>.

**Comment by KatiePUX** [comment]
Tickets 114, 157, 178, 149 linked - let team Kai know when you are working on these so they can make adjustments when required
